conans/client/conf/__init__.py

```diff
@@ -94,7 +94,7 @@
                     "10", "10.1", "10.2", "10.3", "10.4", "10.5",
                     "11", "11.1", "11.2", "11.3", "11.4",
                     "12", "12.1", "12.2", "12.3",
-                    "13", "13.1", "13.2"]
+                    "13", "13.1", "13.2", "14"]
```

Comment:
```suggestion
                    "13", "13.1", "13.2", 
                    "14", "14.1"]
```
It will be better when minor versions are added, also I think the first release will actually be 14.1?

Reply:
@memsharded Oh, you are right. Following GCC release policy the first stable version should be 14.1 not 14